sceptre/plan/plan.py

```diff
@@ -394,3 +394,10 @@ def drift_show(self, *args) -> Dict[Stack, str]:
         """
         self.resolve(command=self.drift_show.__name__)
         return self._execute(*args)
+
+    def dump_config(self, *args):
```

Comment:
I don't know why all the commands on this module just take arbitrary `*args`. Most of those commands don't actually take _any_ arguments, but in this case, you do require one. I'd suggest you explicitly define the argument you need here, just like you did on actions.py.